public/docs/ts/latest/guide/reactive-forms.jade

```diff
@@ -0,0 +1,1013 @@
+  based on the _QuickStart seed_.
+
+.l-main-section
+a#data-model
+:marked
+  ## Create a data model
+  The focus of this guide is a reactive forms component that edits a hero.
+  You'll need a `hero` class and some hero data.
+  Create a new `data-model.ts` file in the `app` directory and copy the content below into it.
+
++makeExample('reactive-forms/ts/app/data-model.ts', '','app/data-model.ts')(format=".")
+
+:marked
+  The file exports two classes and two constants. The `Address` 
+  and `Hero` classes define the application _data model_. 
+  The `heroes` and `states` constants supply the test data. 
+
+.l-main-section
+a#create-component
+:marked
+  ## Create a _reactive forms_ component
+  Make a new file called 
+  `hero-detail.component.ts` in the `app` directory and import these symbols:
+
++makeExample('reactive-forms/ts/app/hero-detail-1.component.ts', 'imports','app/hero-detail.component.ts')(format=".")
+
+:marked
+  Now enter the `@Component` decorator that specifies the `HeroDetailComponent` metadata:
+
++makeExample('reactive-forms/ts/app/hero-detail.component.ts', 'metadata','app/hero-detail.component.ts (excerpt)')(format=".")
+
+:marked
+  The `moduleId: module.id` lets you use 
+  [component-relative paths](./cookbook/component-relative-paths.html) in file URLs 
+  such as when specifying the `templateUrl`.
+
+  Next, create an exported `HeroDetailComponent` class with a `FormControl`. 
+  
+
++makeExample('reactive-forms/ts/app/hero-detail-1.component.ts', 'v1','app/hero-detail.component.ts (excerpt)')(format=".")
+
+:marked
+  Here you are creating a `FormControl` called `name`.
+  It will be bound in the template to an HTML `input` box for the hero name. 
+
+  A `FormControl` constructor accepts three, optional arguments: 
+  the initial data value, an array of validators, and an array of async validators.
+  
+  This simple control doesn't have data or validators.
+  In real apps, most form controls have both. 
+
+.l-sub-section
+  :marked
+    This guide touches only briefly on `Validators`. For an in-depth look at them,
+    read the [Form Validation](../cookbook/form-validation.html) cookbook.
+
+.l-main-section
+a#create-template
+:marked
+  ## Create the template
+
+  
+  Now create the component's template, `app/hero-detail.component.html`, with the following markup.
+
++makeExample('reactive-forms/ts/app/hero-detail-1.component.html', 'simple-control','app/hero-detail.component.html')(format=".")
+
+.l-sub-section
+  :marked
+    Disregard the `form-control` _CSS_ class. It belongs to the 
+    <a href="http://getbootstrap.com/" target="_blank" title="Bootstrap CSS">Bootstrap CSS library</a>,
+    not Angular.
+    It _styles_ the form but in no way impacts the logic of the form.
+
+a#import
+:marked
+  ## Import the _ReactiveFormsModule_
+
+  The HeroDetailComponent template uses `formGroup` and `formControlName` 
+  directives from the `ReactiveFormsModule`.
```

Comment:
Now that we've added the `formControl` directive, this needs to be updated right?  At this point, we are not yet using those.